web/modules/custom/asset_release/asset_release.module

```diff
@@ -115,47 +116,32 @@ function asset_release_rdf_entity_update(EntityInterface $entity) {
 /**
  * Implements hook_preprocess_HOOK().
  */
-function asset_release_preprocess_rdf_entity(&$variables) {
-  /** @var \Drupal\rdf_entity\RdfInterface $rdf_entity */
-  $rdf_entity = $variables['rdf_entity'];
-  if ($rdf_entity->bundle() !== 'asset_release') {
+function asset_release_preprocess_rdf_entity(array &$variables): void {
+  $release = $variables['rdf_entity'];
+  if (!$release instanceof AssetReleaseInterface) {
     return;
   }
-  $variables['full_label'] = new TranslatableMarkup('@label @ver', ['@label' => $rdf_entity->label(), '@ver' => $rdf_entity->field_isr_release_number->value]);
+  $variables['full_label'] = new TranslatableMarkup('@label @version', [
+    '@label' => $release->label(),
+    '@version' => $release->getVersion(),
+  ]);
 
-  /** @var \Drupal\rdf_entity\Entity\Rdf $parent_solution */
-  $parent_solution = \Drupal::service('asset_release.relations')->getReleaseSolution($rdf_entity);
-
-  // Mark the release as latest.
-  $is_latest_release = FALSE;
-  if ($variables['view_mode'] === 'compact') {
-    // @see \Drupal\asset_release\Controller\AssetReleaseController::overview()
-    $is_latest_release = !empty($rdf_entity->is_latest_release);
-  }
-  elseif ($parent_solution) {
-    // @todo Find a better way of marking the latest release that doesn't
-    //   involve performing a database query in the theme layer.
-    // @see https://citnet.tech.ec.europa.eu/CITnet/jira/browse/ISAICP-3101
-    $latest_release = solution_get_latest_release($parent_solution);
-    if ($latest_release && $latest_release->id() === $rdf_entity->id()) {
-      $is_latest_release = TRUE;
-    }
-  }
-
-  $variables['is_latest_release'] = $is_latest_release;
+  $solution = $release->getSolution();
+  $is_latest_release = $release->isLatestRelease();
+  $variables['is_latest_release'] = $release->isLatestRelease();
```

Comment:
We can already use `$is_latest_release` on the second line, no need to call `$release->isLatestRelease()` again.